src/foundry/common/utils/iterable-weak-set.d.mts

```diff
@@ -9,7 +9,7 @@ declare class IterableWeakSet<T extends WeakKey> extends WeakSet<T> {
   /**
    * The backing iterable weak map.
    */
-  #map: IterableWeakMap<T, T>;
+  private map: IterableWeakMap<T, T>;
```

Comment:
This is incorrect; frankly, we should just remove the property - the # syntax is a hard private that works even at runtime, while TS private will just throw you compiler warnings.

Reply:
Removing it is definitely a better options, thanks